libraries/api/test/logging_test.js

```diff
@@ -45,9 +45,10 @@ suite(testing.suiteName(), function() {
     method: 'get',
     route: '/require-no-scopes',
     name: 'requireNoScopes',
-    title: 'Requre no scopse',
+    title: 'Require no scopes',
```

Comment:
wow, that was quite the typo-laden string before!

Reply:
```
8026691dab (Dustin J. Mitchell 2019-04-12 21:49:35 +0000  30)     title: 'Requre some scopse',
```
classic!